webui/src/main/java/org/polypheny/db/webui/models/requests/QueryExplorationRequest.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright 2019-2024 The Polypheny Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.polypheny.db.webui.models.requests;
-
-import lombok.experimental.SuperBuilder;
-import lombok.extern.jackson.Jacksonized;
-
-@SuperBuilder
-@Jacksonized
-public class QueryExplorationRequest extends UIRequest {
```

Comment:
This class should be deleted in the [UI](https://github.com/polypheny/Polypheny-UI/blob/dbf2147d6055facf765ba5c3e90ab213809f90c9/src/app/models/ui-request.model.ts#L98) as well.